CHANGES.rst

```diff
@@ -32,7 +32,7 @@ esa.hubble
 - Refactored query_criteria to use ehst.archive table therefore making the query
   a lot faster. [#2524]
 
-- Update TAP url to avoid 301 HTTPError. [#2567]
+- Update to TAP url to query data and download files, aligned with the new eHST Science Archive. [#2567][#2597]
```

Comment:
more changelog entries are needed for this PR to cover the various changes, see the comments below.

Reply:
Changelog entries added. Please let me know if you think more are required.